src/middleware.ts

```diff
@@ -31,6 +31,30 @@ const handleRedirects = (request: NextRequest) => {
 type PathWithTrailingSlash = `${string}/`;
 
 const REDIRECTS: {from: PathWithTrailingSlash; to: string}[] = [
+  {
```

Comment:
What should we keep from these duplicates (6 entries, 3 actual redirects)?
@lizokm  @stephanie-anderson